run_sets/core.yml

```diff
@@ -0,0 +1,12 @@
+# These features must be runned to initialize the server
+- features/setup_user_and_proxy.feature
+- features/mainpage.feature
+# smdba before we start to sync channels
+- features/smdba.feature
```

Comment:
smbda is not core

If we take this approach with multiples file.yml, we need to consider:
- [ ] modify the rakefile (output.html) is hardocded and will be overwritten by each .yaml (create dyn one)
- [ ] we will end up with having 2 output.html or more, which for reviewing is a big change, ( in past we have only 1 file.html)